src/components/segmentedControl/segmentedControl.svelte

```diff
@@ -121,8 +121,9 @@
     --leo-control-item-icon-gap: var(--leo-spacing-m);
     --leo-control-item-font: var(--leo-font-components-button-default);
 
-    display: flex;
-    max-width: max-content;
+    display: grid;
+    grid-auto-flow: column;
+    max-width: var(--leo-segmented-control-max-w, max-content);
```

Comment:
@fallaciousreasoning and @petemill, or do we want to simply remove the `max-content` altogether?

Reply:
I think it's probably safe to remove?